consensus/consensus.md

```diff
@@ -132,51 +145,77 @@ GENESIS_VERSION: 2
  -------------- ------
 | prev_hash    | 32   |
  -------------- ------
-| root_hash    | 32   |
- -------------- ------
-| trees        |      |
+| state_hash   | 32   |
```

Comment:
These serializations are redundant with serializations.md as amended in https://github.com/aeternity/protocol/pull/92/files so may need fine-tuning after merge.